linux_sql/sql/queries.sql

```diff
@@ -0,0 +1,25 @@
+select cpu_number,
+       id,
+       total_mem
+from host_info
+group by cpu_number,id,total_mem
+order by total_mem desc;
+
+
+select id as "host_id",hostname,aa.ts,aa.avg
+from
+(select  round5(host_usage."timestamp") as ts,CAST((((CAST(RTRIM (total_mem, 'K') as decimal) - memory_free)/CAST(RTRIM (total_mem, 'K') as decimal))*100) as INT) as "avg",host_id as x
+from host_usage
+join host_info
+on host_info.id = host_usage.host_id
+group by ts,avg,host_id
+order by ts
+) aa
+inner join  host_info on id = aa.x
```

Comment:
This is very overcomplicated. Should be able to do this in 4 lines with select, from, group by, and order by.